next.config.js

```diff
@@ -1,5 +1,76 @@
 const withPlugins = require('next-compose-plugins');
 const withOptimizedImages = require('next-optimized-images');
 const withFonts = require('next-fonts');
+const SentryWebpackPlugin = require('@sentry/webpack-plugin');
 
-module.exports = withPlugins([[withOptimizedImages], [withFonts]]);
+const {
+  NEXT_PUBLIC_SENTRY_DSN: SENTRY_DSN,
+  SENTRY_ORG,
+  SENTRY_PROJECT,
+  SENTRY_AUTH_TOKEN,
+  NODE_ENV,
+  VERCEL_GITHUB_COMMIT_SHA,
+} = process.env;
+
+const COMMIT_SHA = VERCEL_GITHUB_COMMIT_SHA;
+
+process.env.SENTRY_DSN = SENTRY_DSN;
+
+const nextConfig = {
+  env: {
+    // This is not safe but who cares
+    airtableKey: 'keyHw6XELHor5g2uE',
+  },
+  serverRuntimeConfig: {
+    rootDir: __dirname,
+  },
+  webpack: (config, options) => {
+    // In `pages/_app.js`, Sentry is imported from @sentry/browser. While
+    // @sentry/node will run in a Node.js environment. @sentry/node will use
+    // Node.js-only APIs to catch even more unhandled exceptions.
+    //
+    // This works well when Next.js is SSRing your page on a server with
+    // Node.js, but it is not what we want when your client-side bundle is being
+    // executed by a browser.
+    //
+    // Luckily, Next.js will call this webpack function twice, once for the
+    // server and once for the client. Read more:
+    // https://nextjs.org/docs/api-reference/next.config.js/custom-webpack-config
+    //
+    // So ask Webpack to replace @sentry/node imports with @sentry/browser when
+    // building the browser's bundle
+    console.log('HEREREREER');
+    if (!options.isServer) {
+      // eslint-disable-next-line no-param-reassign
+      config.resolve.alias['@sentry/node'] = '@sentry/browser';
+    }
+
+    // When all the Sentry configuration env variables are available/configured
+    // The Sentry webpack plugin gets pushed to the webpack plugins to build
+    // and upload the source maps to sentry.
```

Comment:
Why are we uploading the source maps to sentry?

Reply:
so sentry can parse it and show the errors?